inception/inception-ui-core/src/main/java/de/tudarmstadt/ukp/clarin/webanno/ui/core/page/ApplicationPageBase.html

```diff
@@ -30,7 +30,7 @@
 </head>
 <body data-bs-no-jquery="data-bs-no-jquery">
   <div>
-    <img id="spinner" src="images/spinner.gif" />
+    <img id="spinner" alt="" src="images/spinner.gif" />
```

Comment:
Isn't adding an empty `alt` attribute like cheating?

Reply:
From https://www.w3.org/WAI/tutorials/images/: "[Decorative images](https://www.w3.org/WAI/tutorials/images/decorative/): Provide a null text alternative (alt="") when the only purpose of an image is to add visual decoration to the page, rather than to convey information that is important to understanding the page."

If the image is not a decorative image, a reasonable text would be better.

Reply:
The image indicates if an AJAX request to the server is in progress.

Reply:
Then the question is whether some text would be helpful for a blind person. Personally I think no text would be acceptable. Overwise "busy" or a similar text could be used, but that needs localisation of course.

Reply:
I was wondering about localization as well - that would also affect the `lang` tag on the page. But I think fixing such issues should at the moment be considered without localization since we don't really yet support any language but English. When/if localization is addressed, this would need to be revisited.